source/includes/common-minio-ad-ldap-params.rst

```diff
@@ -0,0 +1,129 @@
+.. Descriptions for External Identity Management using an LDAP Provider   
+   Used in the following files:                                                                
+   - /source/reference/minio-mc/mc-idp-ldap-add.rst
+   - /source/reference/minio-mc/mc-idp-ldap-update.rst
+
+   Does not include ALIAS, as the example differs between add and update
+
+.. start-minio-ad-ldap-params
```

Comment:
Huh. This is a very clever way of doing common params in an includes and I'm glad it worked.

Reply:
I was looking at the overwhelmingly identical text in `add` and `update` and went NOPE. Had to be a separate file however, because All The Includes!